File: src/csharp/Intel/Iced/Intel/MasmFormatterInternal/Registers.cs
// SPDX-License-Identifier: MIT
// Copyright (C) 2018-present iced project and contributors

#if MASM
using Iced.Intel.FormatterInternal;

namespace Iced.Intel.MasmFormatterInternal {
	static class Registers {
#pragma warning disable CS0618 // Type or member is obsolete
		public const Register Register_ST = Register.DontUse0;
#pragma warning restore CS0618 // Type or member is obsolete
		public static readonly FormatterString[] AllRegisters = RegistersTable.GetRegisters();
	}
}
#endif

File: src/java/iced-x86/src/main/java/com/github/icedland/iced/x86/fmt/masm/Registers.java
// SPDX-License-Identifier: MIT
// Copyright (C) 2018-present iced project and contributors

package com.github.icedland.iced.x86.fmt.masm;

import com.github.icedland.iced.x86.Register;
import com.github.icedland.iced.x86.internal.fmt.FormatterString;
import com.github.icedland.iced.x86.internal.fmt.RegistersTable;

final class Registers {
	@SuppressWarnings("deprecation")
	public static final int REGISTER_ST = Register.DONTUSE0;
	public static final FormatterString[] allRegisters = RegistersTable.getRegisters();
}
